plugins/netskope/icon_netskope/actions/get_single_user_confidence_index/schema.py

```diff
@@ -0,0 +1,104 @@
+# GENERATED BY KOMAND SDK - DO NOT EDIT
+import insightconnect_plugin_runtime
+import json
+
+
+class Component:
+    DESCRIPTION = "Get the confidence index for a user"
+
+
+class Input:
+    FROMTIME = "fromTime"
+    USERNAME = "username"
+    
+
+class Output:
+    CONFIDENCES = "confidences"
+    USERID = "userId"
+    
+
+class GetSingleUserConfidenceIndexInput(insightconnect_plugin_runtime.Input):
+    schema = json.loads("""
+   {
+  "type": "object",
+  "title": "Variables",
+  "properties": {
+    "fromTime": {
+      "type": "integer",
```

Comment:
Can we change this to Date, I think UI allows user to pick if it is a date. Epoch time is hard. But make sure you put the timezone